src/rankcorr.jl

```diff
@@ -18,6 +18,16 @@ corspearman(x::RealVector, Y::RealMatrix) = cor(tiedrank(x), mapslices(tiedrank,
 
 corspearman(X::RealMatrix) = (Z = mapslices(tiedrank, X, 1); cor(Z, Z))
 
+"""
+    corspearman(x, y)
+
+### Args:
+* `x`: Can be any one of `RealVector` or `RealMatrix`.
+* `y`: Can be any one of `RealVector` or `RealMatrix`.
+
+Spearman’s rank correlation. Here, `x` and `y` can be either real vectors or matrices. When `x` and `y` are matrices, it computes the correlations between their columns (pairwisely).
```

Comment:
I don't think "pairwisely" is an English word. :-)

I also don't think you should repeat "`x` and `y` can be either real vectors or matrices": just say what happens for each type. And what happens when passing a vector and a matrix?